desktop/libs/notebook/src/notebook/templates/editor_components.mako

```diff
@@ -578,7 +578,12 @@ ${ sqlSyntaxDropdown.sqlSyntaxDropdown() }
     </div>
 
     <div data-bind="visible: result.hasResultset() && status() == 'available' && result.data().length == 0 && result.fetchedOnce() && result.explanation().length <= 0, css: resultsKlass" style="display:none;">
-      <pre class="margin-top-10 no-margin-bottom"><i class="fa fa-check muted"></i> ${ _("Done. 0 results.") }</pre>
+      <!-- ko if: result.hasMore -->
+        <pre class="margin-top-10 no-margin-bottom"><i class="fa fa-spin fa-spinner"></i> ${ _('Loading...') }</pre>
```

Comment:
Could we avoid the `<pre class="margin-top-10 no-margin-bottom">` duplication?

Reply:
Actually No , I tried that already, its been css classes have tight coupling with <pre> and <i> so the css wont work when i split it and cause height mismatch like below
So i happened not to touch the logic of seperating it.
<img width="1037" alt="Screenshot 2021-06-09 at 10 36 27 AM" src="https://user-images.githubusercontent.com/18462803/121296594-8dc2a180-c90e-11eb-9eda-4082561fc0c1.png">

Reply:
You should be able to move the if inside the pre i.e.

```
<pre ...>
  <!-- ko if: result.hasMore --> 
    <i class="fa fa-spin" ....>...
  <!-- /ko -->
  <!-- ko ifnot: result.hasMore -->
    <i class="fa fa-check" ...>...
  <!-- /ko -->
</pre>
```

Reply:
Seems like ko comment messes with the css definition, all good with the duplication from my side.

Reply:
as discussed when i preceed the ko if insid pre its considering ko comment and the height of the div is pretty large causing the looks bad.
Keeping the If as same.